### PR TITLE
Added fsi_file_url twig function and additional bundle configuration

### PR DESCRIPTION
**Pull request by chives**

### CODE REVIEW

File: Exception/Uploadable/InvalidFilesystem.php [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FSi\Bundle\DoctrineExtensionsBundle\Exception\Uploadable;
+
+use FSi\DoctrineExtensions\Uploadable\Exception\RuntimeException;
+
+class InvalidFilesystem extends RuntimeException
```
Comment (bartekgruszka) [inline Exception/Uploadable/InvalidFilesystem.php]: Exception class name should always ends with 'Exception', so `InvalidFilesystemException`

**Comment by bartekgruszka** [comment]
:+1:

**Comment by JarekW** [comment]
:+1:

**Comment by chives** [comment]
@fsi-open updated. Please review again

**Comment by rn0** [comment]
:+1:

**Pull request merged**